apps/frontend/src/utils/graphql/rsk/operations/getV2StakingWithdrawns.graphql

```diff
@@ -0,0 +1,27 @@
+query getV2StakingWithdrawns(
```

Comment:
```suggestion
query getStakingWithdraws(
```
Use `getStakingWithdraws` or `getStakingWithdrawals` (the same as for the component and the folder itself).